src/soar_sdk/cli/init/cli.py

```diff
@@ -268,11 +273,21 @@ def convert_connector_to_sdk(
     # Convert the main module path to the SDK format, but save a reference to the original
     app_meta.main_module = "src.app:app"
 
+    app_python_versions = app_meta.python_version
+    enforced_python_versions = _default_python_versions()
```

Comment:
I don't see any future in which the "default" shouldn't be the same as "all", since the only driver for Python version support is SOAR version support, and we want the SDK to drive SOAR compatibility
```suggestion
    enforced_python_versions = PythonVersion.all()
```